components/sup/src/manager/commands.rs

```diff
@@ -226,14 +227,12 @@ pub fn service_load(
     opts: protocol::ctl::SvcLoad,
 ) -> NetResult<()> {
     let ident: PackageIdent = opts.ident.clone().ok_or_else(err_update_client)?.into();
-    let bldr_url = opts
-        .bldr_url
-        .clone()
-        .unwrap_or_else(|| protocol::DEFAULT_BLDR_URL.to_string());
+    let bldr_url = opts.bldr_url.clone().unwrap_or_default();
```

Comment:
Was this a mistake? I don't think we need to be modifying url stuff when making this channel change.

Reply:
Yeah, I'm not sure how that got in there. Reverted in https://github.com/habitat-sh/habitat/pull/6074/commits/8eee5cee3788d6f029ec591ec7394e98ef008ae8